.github/workflows/_release.yml

```diff
@@ -42,3 +46,37 @@ jobs:
       - run: zig build -Dtarget=aarch64-windows
       - run: zip -j fastfec-windows-aarch64-${{ inputs.version }}.zip zig-out/bin/fastfec.exe
       - run: mv zig-out/lib/fastfec.dll libfastfec-windows-aarch64-${{ inputs.version }}.dll
+      - uses: actions/upload-artifact@v2
+        with:
+          path: |
+            fastfec-linux-x86_64-${{ inputs.version }}.zip
+            libfastfec-linux-x86_64-${{ inputs.version }}.so
+            fastfec-linux-aarch64-${{ inputs.version }}.zip
+            libfastfec-linux-aarch64-${{ inputs.version }}.so
+            fastfec-macos-x86_64-${{ inputs.version }}.zip
+            libfastfec-macos-x86_64-${{ inputs.version }}.dylib
+            fastfec-macos-aarch64-${{ inputs.version }}.zip
+            libfastfec-macos-aarch64-${{ inputs.version }}.dylib
+            fastfec-windows-x86_64-${{ inputs.version }}.zip
+            libfastfec-windows-x86_64-${{ inputs.version }}.dll
+            fastfec-windows-aarch64-${{ inputs.version }}.zip
+            libfastfec-windows-aarch64-${{ inputs.version }}.dll
+            libfastfec-${{ inputs.version }}.wasm
+      - uses: softprops/action-gh-release@v1
+        if: inputs.uploadRelease
+        with:
+          tag_name: ${{ inputs.version }}
+          files: |
+            fastfec-linux-x86_64-${{ inputs.version }}.zip
+            libfastfec-linux-x86_64-${{ inputs.version }}.so
+            fastfec-linux-aarch64-${{ inputs.version }}.zip
+            libfastfec-linux-aarch64-${{ inputs.version }}.so
+            fastfec-macos-x86_64-${{ inputs.version }}.zip
+            libfastfec-macos-x86_64-${{ inputs.version }}.dylib
+            fastfec-macos-aarch64-${{ inputs.version }}.zip
+            libfastfec-macos-aarch64-${{ inputs.version }}.dylib
+            fastfec-windows-x86_64-${{ inputs.version }}.zip
+            libfastfec-windows-x86_64-${{ inputs.version }}.dll
+            fastfec-windows-aarch64-${{ inputs.version }}.zip
+            libfastfec-windows-aarch64-${{ inputs.version }}.dll
+            libfastfec-${{ inputs.version }}.wasm
```

Comment:
This is fine, but you might be able to cut down on repetition here and above in the build step by matrixing on the target and artifact file names.

Reply:
Ok, trying out the matrix strategy. Should rapidly speed up the GH actions too